SU-HKKU/cardinal_service_profile
# CSD-486 Add Favorites for resources

# READY FOR REVIEW

# Summary
- Add favorite button to resources

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. `blt drupal:sync --site=cardinalservice`
1. go to the `/resources` page
1. favorite some resources
1. go to your dashboard
1. see that you have some listed as saved.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
